packages/theme-default/src/client/styles/global.css

```diff
@@ -265,14 +265,17 @@ a.__vbk__.header-anchor:focus {
 
 ul.__vbk__,
 ol.__vbk__ {
-  list-style: square;
   margin: 1.15rem 0;
   line-height: 1.8;
+  padding-left: 1.15rem;
+}
+
+ul.__vbk__ {
+  list-style: square;
 }
 
-ul.__vbk__,
 ol.__vbk__ {
-  padding-left: 1.15rem;
```

Comment:
I moved this to the already existing selector above that combines `ul` and `ol` styles.